packages/rest/tests/did.test.ts

```diff
@@ -0,0 +1,50 @@
+import type { Agent, DidResolutionResult } from '@aries-framework/core'
+import type { Server } from 'net'
+
+import request from 'supertest'
+
+import { startServer } from '../src'
+
+import { getTestAgent, getTestDidRecord, objectToJson } from './utils/helpers'
+
+describe('DidController', () => {
+  let app: Server
+  let aliceAgent: Agent
+  let didRecord: DidResolutionResult
+
+  beforeAll(async () => {
+    aliceAgent = await getTestAgent('Connection REST Agent Test Alice', 3999)
+    app = await startServer(aliceAgent, { port: 3000 })
+    didRecord = await objectToJson(getTestDidRecord())
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('Get did resolution result by did', () => {
+    test('should return did resolution record', async () => {
+      const spy = jest.spyOn(aliceAgent.dids, 'resolve').mockResolvedValueOnce(didRecord)
```

Comment:
We don't have to mock this. The did key value can be resolved without needing a mock

Reply:
Got it